crates/bevy_transform/src/bundles.rs

```diff
@@ -17,25 +19,29 @@ use crate::prelude::{GlobalTransform, Transform};
 ///
 /// [`GlobalTransform`] is the position of an entity relative to the reference frame.
 ///
-/// [`GlobalTransform`] is updated from [`Transform`] by systems in the system set
+/// [`GlobalTransform`] is updated from [`Transform`] by systems in the system set. It is also
+/// automatically inserted whenever [`Transform`] is inserted. This is why this struct has
+/// been deprecated.
+///
 /// [`TransformPropagate`](crate::TransformSystem::TransformPropagate).
 ///
 /// This system runs during [`PostUpdate`](bevy_app::PostUpdate). If you
 /// update the [`Transform`] of an entity in this schedule or after, you will notice a 1 frame lag
 /// before the [`GlobalTransform`] is updated.
 #[derive(Clone, Copy, Debug, Default, Bundle)]
+#[deprecated(
+    since = "0.15.0",
+    note = "Use `Transform` on its own: `GlobalTransform` is now a required component."
```

Comment:
```suggestion
    note = "Use the `Transform` component instead. Inserting `Transform` will now also insert a `GlobalTransform` automatically."
```
Reworded so people don't need to know what a 'required' component is to understand